cmd/cosmosapp.go

```diff
@@ -0,0 +1,39 @@
+package cmd
+
+import (
+	abci "github.com/cometbft/cometbft/abci/types"
+	"github.com/cosmos/cosmos-sdk/codec"
+	"github.com/cosmos/cosmos-sdk/server/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+// CosmosApp implements the common methods for a Cosmos SDK-based application
+// specific blockchain.
+type CosmosApp interface {
```

Comment:
Out of curiosity, why is this interface defined?

Reply:
I kept it by mistake. Removed